includes/class-sensei-utils.php

```diff
@@ -1567,6 +1567,21 @@ public static function user_complete_course( $course_id = 0, $user_id = 0 ) {
 	}
 
 	/**
+	 * Checks if HTML security is enabled or disabled
+	 * @return boolean
+	 */
+	public static function html_security()
```

Comment:
This function will become redundant after we implement `maybe_sanitize`, so I'll re-use it instead